src/structs/Webhook.ts

```diff
@@ -126,7 +126,7 @@ export class Webhook {
           res.sendStatus(204);
         }
       } catch (err) {
-        this.options.error?.(err);
```

Comment:
I'm confused, what exactly is getting thrown from functions that isn't an error? If we're handling all errors with a 500 response, shouldn't they also be treated as an error on the library side? Maybe the correct move here should be to change the type of the error callback from `Error` to `unknown`

Reply:
Are we sure we want to do this? It would be a breaking change. I'm not opposed but a major bump seems unnecessary.

Reply:
Well, this is also a breaking change technically, just more subtle